guides/common/modules/proc_enabling-pull-based-transport.adoc

```diff
@@ -0,0 +1,81 @@
+[id="enabling-pull-based-tranport_{context}"]
+= Enabling Pull-based Transport
+
+.Procedure
```

Comment:
This should be a procedure - a sequence of steps with one result. This is not a procedure or concept. Please, create a procedure with a sequence of steps.

Reply:
Updated procedure